README.md

```diff
@@ -1,66 +1,86 @@
-# duniverse -- the spice of OCaml build life
+# opam-monorepo
 
-This is an experimental vendoring system for Dune.  Not for public consumption
-just yet.
+`opam-monorepo` is an opam plugin designed to assemble standalone dune workspaces
+with your projects and all of their dependencies, letting you build it all from scratch
+using only `dune`.
```

Comment:
Not sure how technical you want to be...
```suggestion
using only `dune` and `ocaml`.
```

Reply:
I'm fine with adding ocaml!